joomla/joomla-cms
# com_installer updatesites view: searchtools ordering and bug corrections

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_installer updatesites view.

Also corrects bug where the folder filter and not selecting the search button, and other small graphical issues.
Also other small code changes/improvements.
###### Before PR

![com_installer-updatesites-beforepr](https://cloud.githubusercontent.com/assets/9630530/12993496/69799486-d112-11e5-91bb-dfa292b46711.png)
###### After PR

![com_installer-updatesites-afterpr](https://cloud.githubusercontent.com/assets/9630530/12993502/6eb83b00-d112-11e5-9421-ecdb178c7db5.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Extensions -> Manage -> Update Sites
3. Check the ordering select box and the filter are working fine.
